### ISSUE TITLE
[Select] Improve keyboard management

### ISSUE THREAD

**Issue by jeremie-lucca** [LuccaSA/lucca-front#3181]
Our new select panel all is not fully accessible as some actions cannot be triggered by keyboard.
We need to improve focus strategy to allow keyboard selection on : 

- [x] Select all (Multi)
- [x] Group selection (Multi)
- [x] Add option (Simple & Multi)
- [x] Former employees option (Simple & Multi)
